cpp/include/Ice/StreamHelpers.h

```diff
@@ -10,7 +10,11 @@
 #include <iterator>
 #include <ostream>
 
-#if __has_include(<span>)
+#if defined(_MSC_VER) // workaround for MSVC bug
```

Comment:
MSVC in C++ 17 mode:
 - `__has_include(<span>)` succeeds (it should not since `<span>` is c++20-only)
 - then we include `<span>`
 - the `<span>` code complains it's C++20-only